R/run_stress_test.R

```diff
@@ -92,10 +92,7 @@ run_stress_test <- function(asset_type,
     iter_var = iter_var
   )
 
-  args_tibble <- tibble::as_tibble(args_list) %>%
-    dplyr::mutate(iter_var = .env$iter_var)
-
-  st_results_list <- purrr::map(1:nrow(args_tibble), run_stress_test_iteration, args_tibble = args_tibble)
+  st_results_list <- run_stress_test_iteration(args_list)
```

Comment:
All friends of `run_stress_test()` now input the same single argument `args_list`. That list abstracts the details of which arguments it holds, and make it easy to add, remove, and change the arguments it contains.